doc/source/ray-core/tasks/using-ray-with-gpus.rst

```diff
@@ -123,4 +123,16 @@ This also lets the multi-node-type autoscaler know that there is demand for that
     :start-after: __accelerator_type_start__
     :end-before: __accelerator_type_end__
 
-See ``ray.util.accelerators`` for available accelerator types. Current automatically detected accelerator types include Nvidia GPUs.
+See ``ray.util.accelerators`` for available accelerator types. Current automatically detected accelerator types include Nvidia GPUs, AWS-neuron-cores.
+
+AWS Neuron Core Accelerator
+---------------------------
+.. note::
+
+  Ray supports a heterogeneous cluster of GPUs and NeuronCores but doesn't allow specifying resources requirements of
+  ```num_gpus`` and ``num_neuron_cores`` for task or actor.
```

Comment:
for clarification, does it mean we can't specifying resource requirements of GPU and neuron_cores together?

Reply:
That's right, because a machine cannot have CUDA/GPU and NeuronCores together to execute a task/actor and we fail on validation.